git/exercise-sample-code/gallery.js

```diff
@@ -0,0 +1 @@
+gallery.js
```

Comment:
What's the point of this file?

Reply:
was going to be used (so it was a place holder).  Removed.